backend/spec/controllers/spree/admin/resource_controller_spec.rb

```diff
@@ -146,6 +153,21 @@ def model_class
     it 'destroys the resource' do
       expect { subject }.to change { Widget.count }.from(1).to(0)
     end
+
+    context 'failure' do
+      let(:widget) { Widget.create!(name: 'undestroyable') }
+      let(:params) { { id: widget.id } }
+
+      context 'js format' do
+        subject { delete :destroy, params: params, format: 'js' }
+
+        it 'responses with error message' do
```

Comment:
Can you change `responses` for `responds`? Thanks!

Reply:
@aitbw good catch!

Reply:
Done